SalesforceCommerceCloud/pwa-kit
# Social Login Redirect Page

# Description

In Social Login, users are redirected to a 3rd party IDP site to complete login. When they are redirected back to the storefront site, there will be a redirect page with a loading state until the user is successfully logged in.

Figma Prototype [here](https://www.figma.com/proto/80M7MTsZT6Ipu8xAGi6qUK/%5B254%5D-Passwordless-and-Social-Login?page-id=&node-id=7386-14513&node-type=canvas&viewport=1300%2C-1369%2C0.25&t=y1BWmAc1rMMzIloo-1&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=7238%3A44162&show-proto-sidebar=1)



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Add a new page `app/pages/social-login-redirect` that will be the redirect page in the social login flow

# How to Test-Drive This PR

To see the redirect page:
- Go to https://wasatch-mrt-social-login-poc.mrt-storefront-staging.com/social-callback
- Verify that it looks correct in both the mobile and desktop view

To see the redirect page within the social login flow:
- Go to https://wasatch-mrt-social-login-poc.mrt-storefront-staging.com/login
- Click on `Google` and see the login flow proceed

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
